packages/core/test/integration/model/findAll/groupedLimit.test.js

```diff
@@ -117,7 +117,7 @@ if (current.dialect.supports['UNION ALL']) {
             }
           });
 
-          it('[Flaky] works with computed order', async function () {
+          it('works with computed order', async function () {
```

Comment:
Isn't this test now testing the same thing as the test below? Different sorting order, but this now also has multiple orders of which 1 is computed.

Reply:
It is. We could just delete this test, it doesn't add anything

Reply:
Let's do that then